department-of-veterans-affairs/next-build
# Vacms 16382 metatag updates

## Description

closes #16382 

## Testing done

1. Manual testing as described in the QA steps.
2. Updated snapshot tests to incorporate "lastUpdated" field.

## Screenshots

Custom tags for "/central-iowa-health-care/events/52265"
<img width="734" alt="Screenshot 2023-12-29 at 2 01 41 PM" src="https://github.com/department-of-veterans-affairs/next-build/assets/3640707/8f9c472a-9391-4daf-ac88-b50709390b23">

Default tags:
<img width="582" alt="Screenshot 2023-12-29 at 2 02 14 PM" src="https://github.com/department-of-veterans-affairs/next-build/assets/3640707/2b1c14c3-5987-4308-ac3e-52b77b656cc6">

## QA steps

1. Go to `/central-iowa-health-care/events/52265` to load the page.
2. Look at the metatag output and see that it matches the screenshot for custom tags.
3. Uncomment the `/central-iowa-health-care/events/52265` URL in the `urlsForBanner` variable in the `src/templates/globals/meta/index.tsx` file to show the iOS banner.
4. See that the iOS metatags now show up. 
5. Change `resource.metatags?.length > 0` to `resource.metatags?.length > 1000` in `src/templates/globals/meta/index.tsx` to get the default metatags to show up. 
6. Compare the metatags you see to the default metatag output screenshot.
7. For the two images added, make sure they resolve at the paths: "/img/design/logo/va-og-twitter-image.png" and "/img/design/logo/va-og-image.png".


```[tasklist]
```
